src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -5267,6 +5267,434 @@ void F(int x = 2, = 3) { }
             Assert.Same(symbol1.ContainingSymbol, symbol2.ContainingSymbol);
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74348")]
+        public void ObjectInitializerIncompleteMemberValueAssignment01()
+        {
+            var source = """
+                public class Thing
+                {
+                    public int Key { get; set; }
+                    public string? Value { get; set; }
+                }
+
+                public class Using
+                {
+                    public static Thing CreateThing(int key, string? value)
+                    {
+                        return new()
+                        {
+                            Key = key,
+                            Value,
+                        };
+                    }
+                }
+                """;
+            var comp = CreateCompilation(source);
+
+            comp.VerifyDiagnostics(
+                // (4,18): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.
```

Comment:
>warning CS8632 [](http://example.com/codeflow?start=27&length=14)

Please get rid of these warnings by removing the nullable annotations. I assume they are not required to observe effect of the changes.

Reply:
Done